content/general/aws-authentication.textile

```diff
@@ -0,0 +1,267 @@
+</a>
+
+2. Click "Create Policy".
+
+3. Click the JSON tab and enter the following JSON to configure the policy:
+
+```[json]
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "AllowInvokeLambdaFunction",
+            "Effect": "Allow",
+            "Action": [
+                "lambda:InvokeAsync",
+                "lambda:InvokeFunction"
+            ],
+            "Resource": [
+                "arn:aws:lambda:<YOUR_AWS_REGION>:<YOUR_AWS_ACCOUNT>:function:<YOUR_FUNCTION_NAME>"
+            ]
+        }
+    ]
+}
+```
+
+**Note:** You will need to replace @<YOUR_AWS_REGION>@, @<YOUR_AWS_ACCOUNT>@, and @<YOUR_FUNCTION_NAME>@ with the AWS region that hosts your AWS Lambda function, your AWS account ID, and your AWS Lambda function name respectively.
+
+4. Click "Next: Tags". You don't need to add any tags.
+
+5. Click "Next: Review".
+
+6. Enter a suitable name for your Policy:
+
+<a href="/images/screenshots/iam/review-and-create-policy.png" target="_blank">
+  <img src="/images/screenshots/iam/review-and-create-policy.png" style="width: 100%" alt="Review and create policy">
+</a>
+
+7. Click "Create Policy".
+
+The policy giving permissions to invoke a Lambda function has been created. You now need to attach it to the role that you'll specify in your Ably integration rule. The next step is to "create the role":#create-role.
+
+h3(#create-sqs-policy). Create a policy for the AWS SQS service
+
+The following steps show you how to create a policy for AWS SQS.
+
+1. In the IAM console sidebar select "Policies":
+
+<a href="/images/screenshots/iam/create-policy.png" target="_blank">
+  <img src="/images/screenshots/iam/create-policy.png" style="width: 100%" alt="Create policy">
+</a>
+
+2. Click "Create Policy".
+
+3. Click the JSON tab and enter the following JSON to configure the policy:
+
+```[json]
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "AllowReadWriteSQS",
+            "Effect": "Allow",
+            "Action": [
+                "sqs:DeleteMessage",
+                "sqs:TagQueue",
+                "sqs:GetQueueUrl",
+                "sqs:ChangeMessageVisibility",
+                "sqs:DeleteMessageBatch",
+                "sqs:SendMessageBatch",
+                "sqs:UntagQueue",
+                "sqs:ReceiveMessage",
+                "sqs:SendMessage",
+                "sqs:ListQueueTags",
+                "sqs:ChangeMessageVisibilityBatch"
+            ],
+            "Resource": [
+                "arn:aws:sqs:<YOUR_AWS_REGION>:<YOUR_AWS_ACCOUNT>:<YOUR_QUEUE_NAME>"
+            ]
+        }
+    ]
+}
+```
+
+**Note:** You will need to replace @<YOUR_AWS_REGION>@, @<YOUR_AWS_ACCOUNT>@, and @<YOUR_QUEUE_NAME>@ with the AWS region that hosts your SQS queue, your AWS account ID, and your SQS queue name respectively.
+
+4. Click "Next: Tags". You don't need to add any tags.
+
+5. Click "Next: Review".
+
+6. Enter a suitable name for your Policy.
```

Comment:
Should Policy be capitalised here? I don't think so, because it's not a product name, and you haven't capitalised it elsewhere (except where AWS capitalises it presumably, in button captions, etc)